content/W2016/W16-event-stephanie-talk.md

```diff
@@ -0,0 +1,36 @@
+Title: Circuit Breaker: How my Love for Writing Brought me to Tech
```

Comment:
Its just the name of the talk she sent us, I don't really know what it means either. Should I change the name of the event to "Stephanie Morillo Talk" ?